src/xrpld/app/paths/detail/StrandFlow.h

```diff
@@ -839,7 +847,7 @@ flow(
             // running debug builds of rippled. While this issue still needs to
             // be resolved, the assert is causing more harm than good at this
             // point.
-            // assert(0);
+            // UNREACHABLE();
```

Comment:
there was commented-out `assert(0)` before; replacing it with an actual `UNREACHABLE()` would change the program behaviour from "we put a comment here" to "we crash", and that's not a good PR for this kind of change.

Reply:
Let me rephrase: It was a commented-out `assert`, and you've replaced it by a commented-out `UNREACHABLE`. My assumption here is that you want to leave this here commented out, so it can be re-enabled in the future when needed. If so, the old `assert(0)` statement would compile fine but the new `UNREACHABLE()` statement would fail to compile as that macro expects at least a name to be provided.